setup.py

```diff
@@ -15,7 +15,7 @@
     ],
     include_package_data=True,
     install_requires=[
-        "click==8.0.*",
+        "click>=8.1.7",
```

Comment:
can you please make it "click==8.1.*" ?
I think it might be better. Making it >= can lead to breaking changes according to semantic versioning

Reply:
thanks for your note, I just changed it accordingly.